frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -198,6 +198,12 @@ const PVCType: React.FC<PVCTypeProps> = ({ state, dispatch }) => {
     TiB: 'TiB',
   };
 
+  // Noobaa expected Ti console standrad is to show TiB
+  const unitConverter = {
```

Comment:
can't we use some generic converter present in OCP shared or public code? I don't think this is the first time implementation.